docs/intro.rst

```diff
@@ -207,7 +207,24 @@ Mapping
 
 
 Enum
-....
+----
+
+Enum object can contain values of different types. Enum check determines whether given value is in the Enum instance.
+
```

Comment:
In my opinion it's sound difficult. If u don't know how rewrite simpler it's okay.

Also, I think we need to change order of this two sentences, because first i wanna know what Enum do and then other information.

Reply:
Np, I'll try to rephrase this.

Reply:
thx